dynamics_class: "dubins_car"
safe_set:
  lo: 
    - -5.0
    - -5.0
    - 0.0
  hi: 
    - 5.0
    - 5.0
    - 6.29

obstacles: []

state_domain:
  lo: 
    - -5.0
    - -5.0
    - 0.0
  hi: 
    - 5.0
    - 5.0
    - 6.29
  resolution:
    - 41
    - 41
    - 41
  periodic_dims: []
  
control_space:
  n_dims: 2
  lo:
    - -1.0
    - 0.0
  hi: 
    - 1.0
    - 0.2

disturbance_space:
  n_dims: 2
  lo:
    - 0.0
    - 0.0
  hi:
    - 0.0
    - 0.0